lib/modules/platform/azure/index.ts

```diff
@@ -766,6 +773,36 @@ export function getIssueList(): Promise<Issue[]> {
   return Promise.resolve([]);
 }
 
+async function createWi(title: string, description: string): Promise<number> {
+  const azureApiworkItemTracking = await azureApi.workItemTrackingApi();
+
+  const patchDoc = [
+    {
+      op: 'add',
+      path: '/fields/System.Title',
+      value: title,
+    },
+    {
+      op: 'add',
+      path: '/fields/System.Description',
+      value: description,
+    },
+  ] as JsonPatchDocument;
```

Comment:
Does this work with `JsonPatchOperation`? `JsonPatchDocument` is an empty interface, so doesn't really give any type help. Whereas `JsonPatchOperation` is defined as:

```ts
/**
 * The JSON model for a JSON Patch operation
 */
export interface JsonPatchOperation {
    /**
     * The path to copy from for the Move/Copy operation.
     */
    from?: string;
    /**
     * The patch operation
     */
    op: Operation;
    /**
     * The path for the operation. In the case of an array, a zero based index can be used to specify the position in the array (e.g. /biscuits/0/name). The "-" character can be used instead of an index to insert at the end of the array (e.g. /biscuits/-).
     */
    path: string;
    /**
     * The value for the operation. This is either a primitive or a JToken.
     */
    value?: any;
}
```